src/bg/on-user-script-xhr.js

```diff
@@ -105,11 +105,11 @@ function open(xhr, d, port, tabUrl) {
   d.responseType && (xhr.responseType = d.responseType);
   d.timeout && (xhr.timeout = d.timeout);
 
-  var hasCookieHeader = false;
+  let hasCookieHeader = false;
   if (d.headers) {
-    for (var prop in d.headers) {
+    for (let prop in d.headers) {
       if (Object.prototype.hasOwnProperty.call(d.headers, prop)) {
-        var propLower = prop.toLowerCase();
+        let propLower = prop.toLowerCase();
```

Comment:
`const`? `propLower` has no other purpose than being the lowerCase version of `prop`!